src/Components/DisplayArea/DisplayArea.js

```diff
@@ -1,6 +1,7 @@
 import './DisplayArea.css';
 import selected from '../../Assets/selected.png';
 import unselected from '../../Assets/unselected.png';
+import ErrorPage from '../ErrorPage/ErrorPage';
 const { v4: uuidv4 } = require('uuid');
```

Comment:
What is this wild thing v4??

Reply:
This creates unique keys for each of my recipe cards! I was running into trouble with Date.now() because it was rendering the same time stamp for multiple cards.